omod/src/main/java/org/openmrs/module/owa/utils/OwaFileNameUtil.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0 + Health disclaimer. If a copy of the MPL was not distributed with
+ * this file, You can obtain one at http://license.openmrs.org
+ */
+package org.openmrs.module.owa.utils;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.commons.lang.StringUtils;
+
+public class OwaFileNameUtil {
+
+	public static String getPassedFileName(String installUrl) {
+		String passedFileName = null;
+		if (installUrl.contains("file_path=")) {
+			passedFileName = StringUtils.substringBetween(installUrl, "file_path=", ".zip");
+		} else {
+			passedFileName = FilenameUtils.getName(installUrl);
+		}
+		return getFileName(passedFileName);
+	}
+
+	public static String getFileName(String passedFileName) {
```

Comment:
Good method names should tell what the method is doing. A name like this does not make much sense with a parameter named passedFileName. As in why get a file name when am already passing it in as a parameter? Look at this from the perspective of one who wants to use your utility method without knowing the implementation details.

Reply:
Changed to method name to be more descriptive i.e. `removeVersionNumber`